templates/sheets/item/taxonomy-config.hbs

```diff
@@ -33,10 +33,16 @@
         <div class="form-fields">
             {{#each resistances as |r|}}
             <label>
-                {{formInput r.field value=r.value type="number"}}
+                {{formInput r.field value=r.value disabled=(eq r.value 4) type="number" dataset=(object damageType=r.id)}}
```

Comment:
I don't love using these logical helpers in the HBS files and would prefer that we prepare the attributes we need for rendering as part of the `resistances` array that is passed into the template.
```suggestion
                {{formInput r.field value=r.value type="number" dataset=r.dataset disabled=r.disabled}}
```